### PR TITLE
SQL-1496: Bring doc to inline with MSFT side and update connection dialog

### PR DESCRIPTION
**Pull request by nbagnard**
Updating the doc to be the same as the one on the MSFT repo and also updating the connection dialog to match the new one with the logging level selector.

### CODE REVIEW

**Review by terakilobyte — APPROVED** [review]
LGTM, I made a few suggestions.

**Review by bucaojit — COMMENTED** [review]
@nbagnard looks like we hit our first snag with uploading pictures already.  The [mongodb_connect_to_data_source.png](https://github.com/mongodb/mongo-powerbi-connector/pull/30/files#diff-1a82b9b526de1cf45442bba98cadbcc160f35dd7f93b1a066e2579e3f4c0b702) image should have the Logging Level entry also, but it seems to me that the connector would need to be added by Microsoft for that to be updated.  Should we remove just that image?

File: mongodb-atlas-sql-interface.md [hunk 1]
```diff
@@ -5,70 +5,102 @@
...
+
    - The **MongoDB URI**. _Required_.   
-     Use the MongoDB URI you obtained in step 1.  Make sure that it does not contain your username and password. URIs containing username and/or passwords will be rejected.
-   -  Your federated **Database** name. _Required_  
-      Use the name of the federated database you obtained in step 1.
-   - **Logging Level**.  _Optional, Default value: INFO_
+     Use the MongoDB URI obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).  Make sure that it doesn't contain your username and password. URIs containing username and/or passwords are rejected.
+   - Your federated **Database** name. _Required_  
+     Use the name of the federated database obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).
+   - The logging level for the underlying ODBC driver.
```
Comment (bucaojit) [inline mongodb-atlas-sql-interface.md:55]:
```suggestion
   - The logging level for the underlying ODBC driver.  _Optional, Default value: INFO_
```
The optional and default value is still relevant.

File: mongodb-atlas-sql-interface.md [hunk 2]
```diff
@@ -5,70 +5,102 @@
...
+1. Select **MongoDB Atlas SQL** from the **Power Query - Choose data source** page.
+2. On the **Connection settings** page, fill in the following values:
+    - The **MongoDB URI**. _Required_.   
+      Use the MongoDB URI obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).  Make sure that it doesn't contain your username and password. URIs containing username and/or passwords are rejected.
+    - Your federated **Database** name. _Required_  
+      Use the name of the federated database obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).
+    - Enter a **Connection name**.
+    - Choose a **Data gateway**.
+    - Enter your Atlas MongoDB Database access username and password and select **Next**.
+
```
Comment (bucaojit) [inline mongodb-atlas-sql-interface.md:88]: Log level information should also be here.  I'm pretty sure the Power BI desktop connection dialogue will match what is in the Online connection settings.

**Comment by nbagnard** [comment]
> @nbagnard looks like we hit our first snag with uploading pictures already. The [mongodb_connect_to_data_source.png](https://github.com/mongodb/mongo-powerbi-connector/pull/30/files#diff-1a82b9b526de1cf45442bba98cadbcc160f35dd7f93b1a066e2579e3f4c0b702) image should have the Logging Level entry also, but it seems to me that the connector would need to be added by Microsoft for that to be updated. Should we remove just that image?

It seems like it. Let me do that.

**Review by bucaojit — APPROVED** [review]
Changes look good 👍

File: mongodb-atlas-sql-interface.md [hunk 3]
```diff
@@ -5,70 +5,102 @@
...
+3. Select **Connect using the Atlas SQL Interface**.
+4. Select **Power BI Connector**.
+5. Copy your federated database name and MongoDB URI. You'll need them in a later step.
+
 > [!NOTE]
 > If some or all of your data comes from an Atlas cluster, you must use MongoDB version 5.0 or greater for that cluster to take advantage of Atlas SQL.
 
-We also recommend to install the MongoDB Atlas SQL ODBC Driver before using the MongoDB Atlas SQL connector.
-
+We also recommend that you install the MongoDB Atlas SQL ODBC Driver before using the MongoDB Atlas SQL connector.
```
Comment (alexiantonino) [inline mongodb-atlas-sql-interface.md]: QQ - is this a "recommendation" or a "requirement"? I am pretty sure we should include this in the note with the requirement of Atlas 5.0. And should we include the the link to the download center?

Comment (nbagnard) [reply mongodb-atlas-sql-interface.md]: The ODBC driver is a requirement. You can install it after the connector. So maybe this is what this was about.
The MongoDB 5.0 requirement comes from our translations being only compatible with 5.0 and later I believe.

**Comment by nbagnard** [comment]
evergreen merge

**Pull request merged**